addons/maintenance/models/maintenance.py

```diff
@@ -95,6 +95,7 @@ def _track_subtype(self, init_values):
     @api.depends('serial_no')
     def _compute_display_name(self):
         for record in self:
+            record.display_name = record.name
             if record.name and record.serial_no:
                 record.display_name = record.name + '/' + record.serial_no
             if record.name and not record.serial_no:
```

Comment:
Thanks for the fix! A small improvement could be done:

```suggestion
        for record in self:
            if record.serial_no:
                record.display_name = record.name + '/' + record.serial_no
            else:
```
since record.name is required and record.name is the default (i.e. no need to assign it twice)

Reply:
thanks @ticodoo ,
i was waiting for the test to get finished before pinging you